Tasks/Ignatchik/Task_5/Task_5_5.py

```diff
@@ -0,0 +1,16 @@
+list_input = [0, 1, 2, 3, 4, 7, 8, 10]
+
+def nums(lst):
+    res = str(lst[0])
+    for n in range(len(lst) - 1):
+        if lst[n] + 1 == lst[n + 1]:
+            if res[-1] != '-':
+                res += '-'
+        else:
+            if f'{lst[n]}' not in res:
+                res += f'{lst[n]}, {lst[n + 1]}'
+            else:
+                res += f'{lst[n + 1]}'
```

Comment:
concatenation AND f-string, you're quite a diplomate)) it would be better to choose a single approach